pkgs/development/python-modules/dateparser/default.nix

```diff
@@ -1,47 +1,41 @@
-{ stdenv, fetchFromGitHub, buildPythonPackage, isPy3k
+{ lib, fetchPypi, buildPythonPackage, isPy3k
 , nose
 , parameterized
 , mock
 , glibcLocales
 , six
 , jdatetime
-, pyyaml
 , dateutil
 , umalqurra
 , pytz
 , tzlocal
 , regex
 , ruamel_yaml }:
+
 buildPythonPackage rec {
   pname = "dateparser";
-  version = "0.6.0";
+  version = "0.7.0";
 
-  src = fetchFromGitHub {
-    owner = "scrapinghub";
-    repo = pname;
-    rev = "refs/tags/v${version}";
-    sha256 = "0q2vyzvlj46r6pr0s6m1a0md1cpg9nv1n3xw286l4x2cc7fj2g3y";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "940828183c937bcec530753211b70f673c0a9aab831e43273489b310538dff86";
   };
 
-  # Replace nose-parameterized by parameterized
-  prePatch = ''
-    sed -i s/nose_parameterized/parameterized/g tests/*.py
-    sed -i s/nose-parameterized/parameterized/g tests/requirements.txt
-  '';
-
-  # Upstream Issue: https://github.com/scrapinghub/dateparser/issues/364
-  disabled = isPy3k;
-
-  checkInputs = [ nose parameterized mock glibcLocales ];
+  checkInputs = [ nose mock parameterized six glibcLocales ];
   preCheck =''
     # skip because of missing convertdate module, which is an extra requirement
     rm tests/test_jalali.py
   '';
 
-  propagatedBuildInputs = [ six jdatetime pyyaml dateutil
-            umalqurra pytz tzlocal regex ruamel_yaml ];
+  propagatedBuildInputs = [
+    # install_requires
+    dateutil pytz regex tzlocal
+    # extra_requires
+    (if isPy3k then null else umalqurra)
```

Comment:
++ lib.optional isPy3k umalqurra

Reply:
I had that in the first place, but I want to somehow make clear it is an extra requirement.

Reply:
Why would it be clearer this way? Simply stating that it is optional (extra_requires) is sufficient.

Reply:
One could think it is required for Python 2. But I'll change it anyway.

Reply:
Actually, it seems like umalqurra is now supported on Python 3. No need for lib.optional.